docs/FCC.md

```diff
@@ -0,0 +1,15 @@
+## FCC Statement
+![FCC Logo](imgs/manual/fcc-logo.png)
+1. This device complies with Part 15 of the FCC Rules. Operation is subject to the following two conditions:
+    - This device may not cause harmful interference.
+    - This device must accept any interference received, including interference that may cause undesired operation.
+2. FCC Radiation Exposure Statement: Mobile device.
```

Comment:
We are an unintentional radiator (we do not have an antenna). I believe 2. should be removed. Here is the FCC guidance on what we need to include: https://www.ecfr.gov/current/title-47/chapter-I/subchapter-A/part-15/subpart-B

Reply:
I've updated that section with what I believe to be the correct information; once I make my next commit I'll ask that you confirm or deny since that is a very lengthy page to parse out valuable info from.